add-on/src/landing-pages/welcome/page.js

```diff
@@ -79,14 +79,14 @@ const renderWelcome = (i18n, peerCount, onOpenWebUi) => {
 
   return html`
     <div class="w-80 flex flex-column justify-center">
-      <div class="mb4 flex flex-column justify-center items-center">
+      <div class="mb3 flex flex-column justify-center items-center">
         ${checkmarkSvg()}
-        <p class="mt2 mb0 f3">${i18n.getMessage('page_landingWelcome_welcome_title')}</p>
+        <p class="mt2 mb0 f3 tc">${renderTranslatedSpans('page_landingWelcome_welcome_peers', [peerCount], 'class="aqua fw6"')}</p>
       </div>
-      <p class="${copyClass}">${renderTranslatedSpans('page_landingWelcome_welcome_peers', [peerCount], 'class="aqua fw6"')}</p>
-      <p class="${copyClass} mb4">${renderTranslatedLinks('page_landingWelcome_welcome_discover', ['https://github.com/ipfs-shipyard/ipfs-companion#features'], `target="_blank" class="${anchorClass}"`)}</p>
-      <div class="mt4 f5 flex justify-center items-center">
-        <button class="pv3 ph4 b navy br2 bn bg-white hover-bg-white-90 pointer" onclick=${onOpenWebUi}>${i18n.getMessage('panel_openWebui')}</button>
+      <div class="mt3 f5 flex justify-center items-center">
+        <button class="pv3 ph4 mh2 b navy br2 bn bg-white hover-bg-white-90 pointer" onclick=${onOpenWebUi}>${i18n.getMessage('panel_openWebui')}</button>
+        <button class="pv3 ph4 mh2 b navy br2 bn bg-white hover-bg-white-90 pointer" onclick=${onOpenWebUi}>Browse Files</button>
+        <button class="pv3 ph4 mh2 b navy br2 bn bg-white hover-bg-white-90 pointer" onclick=${onOpenWebUi}>Explore Peers</button>
```

Comment:
loose idea: would it make sense to use the same labels and icons on buttons as we do in WebUI's left-side menu? (creating visual continuity)

Reply:
Yeah, I was kind of debating this too. Strictly speaking, buttons should have a verb in them (clicking a button = taking an action), BUT we also have the pattern of "status/files/peers" in two other places: the Web UI nav, and in Desktop's cube menu. Also, adding the verbs to the buttons makes them substantially longer, and wrap to two lines at narrower widths. Let's go with the shorter button titles.